apps/podium/pages/comments/index.tsx

```diff
@@ -0,0 +1,36 @@
+import styles from '../index.module.scss';
+import Comment from '../../app/comment/comment';
+import SendComment from '../../components/send-comment/send-comment';
+import styled from 'styled-components';
+
+export function Index() {
+  /*
+   * Replace the elements below with your own.
+   *
+   * Note: The corresponding styles are in the ./index.scss file.
+   */
+  return (
+    <PhoneScreen>
```

Comment:
Could you add another div outside so that the background color is shown across the entire page instead of just the small strip